prov/usnic/src/usdf_cq.h

```diff
@@ -36,6 +36,8 @@
 #ifndef _USDF_CQ_H_
 #define _USDF_CQ_H_
 
+#define SREAD_SLEEP_TIME 5
```

Comment:
nit: `SREAD_SLEEP_TIME_MS` would make it clearer that this time is in milliseconds